### PR TITLE
Admin model and related APIs added

### PR DESCRIPTION
**Pull request by parinayc20**
@shashank1310 also contributed
There is a resume selection problem that needs to be fixed.

### CODE REVIEW

**Comment by namangup** [comment]
Change parent directory name to `admin`
Update Readme (check if some previous api also needs to be updated)

**Comment by parinayc20** [comment]
The parent directory's name can't be changed to admin as it is conflicting with django's admin.

**Comment by namangup** [comment]
In `src/urls.py` check if you can use the URL `admin` for `include(admins.urls)`. Remove the `admin` import from `django.contrib` since we anyway won't be using the django admin panel.
Keep the parent directory name as it is.

**Comment by parinayc20** [comment]
Model Finalized, Readme updated and the issue regarding `admin` resolved.

File: admins/views.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                placement = Placement.objects.get(key=key)
+                placement.delete()
+                return Response(status=status.HTTP_200_OK)
+            except:
+                response = {"message": "Invalid key"}
+                return Response(response, status=status.HTTP_400_BAD_REQUEST)
+        else:
+            return Response(status=status.HTTP_401_UNAUTHORIZED)
+
+
```
Comment (namangup) [inline admins/views.py:156]: Implement authentication check.
Make it a class with a get function.

File: admins/views.py [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+            )
+
+        return response
+    except:
+        return HttpResponse("Page Not Found")
+
+
+
+def exportPlacementData(request, key):
+    try:
```
Comment (namangup) [inline admins/views.py]: Same as above.

**Review by namangup — CHANGES_REQUESTED** [review]

**Comment by parinayc20** [comment]
Made the requested changes.

**Pull request merged**